code/game/gamemodes/wizard/wizard.dm

```diff
@@ -72,3 +73,20 @@
 //returns whether the mob is a wizard (or apprentice)
 /proc/iswizard(mob/living/M)
 	return M.mind && M.mind.has_antag_datum(/datum/antagonist/wizard,TRUE)
+
+/datum/game_mode/wizard/generate_credit_text()
+	var/list/round_credits = list()
+	var/len_before_addition
+

```

Comment:
I feel like we should get a special message if the wizard lost within 5 minutes of first arriving on/at the station's z-level (and an extra special one if they met the above condition and knew the fireball spell (ideally, the message would only trigger if they died quickly and fireballed themselves, but I'm not sure that we have the means to check if a wizard has fireballed themselves (since people with null rods still aren't immune to fireballs that don't directly hit them)).